### ISSUE TITLE
wordpress.tv embed is not expanded

### ISSUE THREAD

**Issue by joyously** [WordPress/twentynineteen#501]
In both SeaMonkey and Firefox, the wordpress.tv embed is not expanded.
When viewing the same page with my own theme, which does nothing in regards to embeds or GB, it is expanded in both browsers.
The test URL is
`https://wordpress.tv/2018/10/14/kjell-reigstad-allan-cole-how-we-made-our-first-gutenberg-powered-theme/`

**Comment by ianbelanger79** [comment]
I can confirm that there is an issue with the video on the test url in Firefox, Microsoft Edge and IE11. I can see the video expanded in all browsers, but the video will not play, it just keeps buffering.

**Comment by joyously** [comment]
Did we just happen to find a bad video?  I got it from a proposed block version of the theme unit test data.

**Comment by kjellr** [comment]
I've tested this in Safari, Firefox, and Chrome (via MacOS 10.14). I embedded via the Classic Editor, and in Gutenberg. In all those cases, it expands and plays just fine on my end. Not sure what's causing this. 

> In both SeaMonkey and Firefox, the wordpress.tv embed is not expanded.

When you're saying it's not expanded, is that in both the editor and on the front end?

**Comment by joyously** [comment]
> When you're saying it's not expanded, is that in both the editor and on the front end?

I'm referring solely to the front end.

**Comment by carolinan** [comment]
I cannot reproduce this on Firefox 63.0.1, IE11, or Microsoft EdgeHTML 17.17134 on Win10 with WP 5.0-beta3-43878

**Comment by joyously** [comment]
Can you remove the `browser issue` label? The issue was not present in a different theme with the same content in the same browsers.

**Comment by kjellr** [comment]
I've removed that label, but I'm still unable to reproduce the issue on any of my browsers/devices. Can you provide screenshots and/or more details on the situation you're seeing?

Here are videos of what I'm doing (using Firefox 63.0.1 in the Classic Editor, as well as in Gutenberg): 

https://cloudup.com/cn1J6KBSWqh

In both cases, the example video appears to expand and play just fine.

**Comment by joyously** [comment]
I downloaded the zip again after your comment, and opened the post with the block embeds in three different themes, in SeaMonkey. In this screenshot, Gutenberg is not active. Even when I activate it and reload the page, the embed still is not expanded (but there is more space for it, due to GB styles).
I did this same thing with Firefox, and get the same result. 
Could it be related to my slow connection? I doubt it, because 2 themes show the video, and one shows just the URL.

![tv-embed-seamonkey_2018-11-12_10-13-15](https://user-images.githubusercontent.com/7957680/48360048-e8ff5b00-e663-11e8-9cdf-5b45b1191457.png)

**Comment by kjellr** [comment]
I'm still unable to reproduce this in any browser I've tried: 

**MacOS 10.14.1, Safari 12.0.1**
<img width="751" alt="safari" src="https://user-images.githubusercontent.com/1202812/49296961-8b527780-f487-11e8-9312-9983d1cbee8b.png">

**MacOS 10.14.1, Chrome 70**
<img width="763" alt="chrome" src="https://user-images.githubusercontent.com/1202812/49296982-9c02ed80-f487-11e8-9f45-f372e8769f7c.png">

**MacOS 10.14.1, Firefox 63.0.3**
<img width="761" alt="ff" src="https://user-images.githubusercontent.com/1202812/49297001-ad4bfa00-f487-11e8-9f8b-edd0acb343e9.png">

**Windows 7, Edge**
<img width="718" alt="edge" src="https://user-images.githubusercontent.com/1202812/49297018-b937bc00-f487-11e8-97af-bc98e3f7a71e.png">

**Windows 7, Chrome 70**
<img width="713" alt="windows7-chrome" src="https://user-images.githubusercontent.com/1202812/49297033-c3f25100-f487-11e8-8d97-d2c802e0465f.png">

**iOS 12.1, Safari**
<img width="375" alt="screen shot 2018-11-30 at 10 05 01 am" src="https://user-images.githubusercontent.com/1202812/49297067-dff5f280-f487-11e8-9196-1cfc0c8f0436.png">

I'm going to close this for now, but if someone else comes across the issue, we'll reopen.

**Comment by joyously** [comment]
I had a thought.
The test data I was using was the XML file for the theme unit test. It contained the post meta fields for the embed already. I'm not sure which editor or theme I used first on this, but I'm guessing that I didn't edit it at all, and only displayed the front end, first in my own theme and then in Twenty Nineteen.
Since Twenty Nineteen is the only theme I have that does something with responsive embeds (meaning that it calls `add_theme_support` for it), surely that is a clue. 
Let me emphasize again that it is not the browser, and it is not the content, and it is not the editor(I didn't edit, and it did it with or without GB). 
All I did was change themes and reload the page, and got different results from them.
